lib/ProductOpener/Attributes.pm

```diff
@@ -632,7 +632,9 @@ sub compute_attribute_ecoscore ($product_ref, $target_lc, $target_cc) {
 			$score = $product_ref->{ecoscore_data}{"scores"}{$target_cc} // 0;
 			$grade = $product_ref->{ecoscore_data}{"grades"}{$target_cc};
 		}
-
+		if (($target_cc eq "world") or (!defined $product_ref->{ecoscore_data}{"scores"}{$target_cc})) {
+			$attribute_ref->{missing} = lang_in_other_lc($target_lc, "ecoscore_warning_transportation_short");
```

Comment:
In fact we use the "missing" field of attributes only when the attribute cannot be computed (e.g. we don't have the ingredients list, so we don't know if there's an allergen).

When we display attributes in the summary, we don't display warnings (like for the Nutri-Score, we don't say that fibers are missing and were not taken into account). We display the warning in the knowledge panel instead.

Otherwise it makes a very big summary:

![image](https://user-images.githubusercontent.com/8158668/212303894-f9018e49-cfbe-44ff-b48b-d5d12ed48eae.png)

Reply:
That's where we should add "The full impact of transportion is not included." I think:

![image](https://user-images.githubusercontent.com/8158668/212304243-89be116b-b07d-48b9-be70-17d15a0ef995.png)

Reply:
I'm unclear on what you are saying here - we are already displaying the "Select a country..." warning here. Are you proposing something in addition to that?

Reply:
@john-gom We can leave the "Select a country" warning as-is.